spec/services/symphony_client_spec.rb

```diff
@@ -203,6 +216,17 @@
         expect(JSON.parse(place_hold_response)).to include 'key' => 'other_hold_key'
       end
     end
+
+    context 'when place hold return records in use error' do
+      let(:item_barcode) { 'records_in_use_barcode' }
+      let(:error_prompt) { { messageList: [{ code: 'hatErrorResponse.116', message: 'Records in use' }] }.to_json }
+
+      it 'keeps trying for 5 seconds max until the record is not in use' do
+        place_hold_response = client.place_hold(patron, user.session_token, item_barcode, hold_args)
+
```

Comment:
Also here, I wonder if we can make the spec actually prove that the loop is running. Maybe like an expectation that `sleep` gets invoked? 🤔

Reply:
I am looking into this

Reply:
explained it to me in slack, yes this is the same as the other stubbed request and_return thing with multiple responses 👍